http/swagger.yml

```diff
@@ -9153,6 +9163,10 @@ components:
           type: string
         messageTemplate:
           type: string
+    SlackNotificationRule:
+      allOf:
+        - $ref: "#/components/schemas/NotificationRuleBase"
+        - $ref: "#/components/schemas/SlackNotificationRuleBase"
```

Comment:
I think @121watts is editing these same schemas at the moment, just a heads up.

Reply:
Good to know